pkg/openstack/cinder.go

```diff
@@ -17,10 +17,6 @@ import (
 
 // ReconcileCinder -
 func ReconcileCinder(ctx context.Context, instance *corev1beta1.OpenStackControlPlane, helper *helper.Helper) (ctrl.Result, error) {
-	if !instance.Spec.Cinder.Enabled {
```

Comment:
Since this is an established design pattern, I acknowledge that we might not have the appetite to change this at this point. The alternative I would suggest is that we instead update the Status of disabled services to report "Ready" with the message "$serviceName is disabled".

That would allow the overall status of the OpenStackControlPlane to move to Ready, while maintaining the established design pattern. It might just be a little confusing for users though if we're reporting Ready for a service that isn't deployed.

Reply:
Yeah, I'll revert this and leave it as is.